src/sentry/seer/anomaly_detection/store_data.py

```diff
@@ -136,7 +153,19 @@ def send_historical_data_to_seer(alert_rule: AlertRule, user: User) -> BaseHTTPR
         base_error_response.status = status.HTTP_408_REQUEST_TIMEOUT
         return base_error_response
 
-    # TODO warn if there isn't at least 7 days of data
+    MIN_DAYS = 7
+    data_start_index, data_end_index = get_start_and_end(historical_data)
+    if data_start_index == -1:
+        resp.status = 202
```

Comment:
Ah ok - RE: comments above, lets just throw an exception if there's not enough data.
This _should_ be a loud failure, rather than a seeming success

Reply:
I think that we do create the alert rule—it just won't detect any anomalies until Seer has at least a week of data. Should we still have an exception in this case?

Reply:
got it, that makes sense!

Commented above - rather than modifying the response and returning the response,
Thoughts on simply returning a boolean for successful backfill or not?

we're not utilizing the rest of the response, and modifying the response is unexpected (for ex. mocking seer response w/ 200, but expecting a 202?)

Reply:
I think we can do that